### ISSUE TITLE
Enhancement: Migrate to Qt6

### ISSUE THREAD

**Issue by feliwir** [variar/klogg#418]
Hello everyone,

i just found this tool as a replacement for **glogg** and want to thank you for your work and effort.
 
Since i use Qt myself at work i have experience of migrating from Qt5 to Qt6 and wanted to ask if there is any interest of me porting this project to Qt6. I know there isn't a "big" benefit from migrating, but i feel that now that Qt6 has matured it would maybe make sense to use the latest version.

Just let me know if a PR addressing this would have a chance of getting approved :smile:

**Comment by feliwir** [comment]
![image](https://user-images.githubusercontent.com/2104576/140071803-541c6193-8f42-47da-9df4-c738b077d123.png)
And it's done. This is how klogg looks with Qt 6.2.1

**Comment by variar** [comment]
Great! I'll have a look at your PR.

**Comment by feliwir** [comment]
Great, thanks :)

**Comment by variar** [comment]
I've integrated PR by @feliwir with some modifications. Klogg can be built with Qt6 now. I'm thinking of 
switching Windows and Mac  packages to Qt6 builds after next stable release. For prebuilt Linux packages we are stuck with older Qt versions that are provided with Ubuntu LTS releases.

**Comment by ilpssun** [comment]
I tried this release build: [klogg-22.06.0.1289-Win-x64-Qt6-setup.exe](https://github.com/variar/klogg/releases/download/v22.06/klogg-22.06.0.1289-Win-x64-Qt6-setup.exe) – and quickly switched back to the Qt5 build.  With Qt6, long lines were terminating prematurely (aproximately at screenwidth).  Also, as far as I recall, the menus were not displayed correctly (i. e. some entries were not visible/rendered outside of the visible area).  I did not do any in-depth testing as the first impression was “not usable” on my Win 10 machine.
